lib/validation.js

```diff
@@ -112,6 +113,12 @@ module.exports = function (yargs, usage, y18n) {
       }
     })
 
+    argv._.forEach(function (key) {
+      if (!commandKeys.hasOwnProperty(key)) {
+        unknown.push(key)
```

Comment:
I really appreciate that this implementation means that we don't need to go back and get new translations from people. Great work.